dnf-behave-tests/common/lib/cmd.py

```diff
@@ -48,6 +48,10 @@ def run_in_context(context, cmd, can_fail=False, **run_args):
 
     if not can_fail and context.cmd_exitcode != 0:
         raise AssertionError('Running command "%s" failed: %s' % (cmd, context.cmd_exitcode))
+    elif expected_exit_code is not None and int(expected_exit_code) != context.cmd_exitcode:
```

Comment:
The conversion to `int` should happen on the input of the step, `run_in_context()` here should be accepting only numbers. Since I'm late to the party: https://github.com/rpm-software-management/ci-dnf-stack/pull/1069